.env.example

```diff
@@ -0,0 +1,5 @@
+export AZURE_TENANT_ID=""
+export AZURE_CLIENT_ID=""
+export AZURE_CLIENT_SECRET=""
+export SUBSCRIPTION_ID=""
+export KUBE_CONFIG_FILE=""
```

Comment:
Don't think we need this for the local dev flow.  We already have a file here: https://github.com/Azure/azure-k8s-metrics-adapter/blob/master/local-dev-values.yaml.example

If you didn't find that file useful or the docs on how to get set up we can discuss and create a seperate PR to improve contribution docs.

Reply:
sure!